src/commands/withdraw.js

```diff
@@ -0,0 +1,59 @@
+const { OOPS_TEXT } = require('../messages')
+
+const WITHDRAW_TEXT = 'Wow. Successful withdrawal.'
+const PROPER_AMOUNT_TEXT = 'You need provide a proper amount to be send.'
+const NO_COMMA_TEXT = 'Please avoid "," in your amount and use "."'
+const NEED_ADDRESS_TEXT = 'Need an address as a third argument'
+const NO_FUNDS = 'You dont have doge to transfer.'
+const NOT_ENOUGH_FUNDS = 'Not enough funds for this transfer. Please add some dogecoins.'
+
+function withdraw(message, dogecoinNode, amount, toAddress) {
+  var amountInt = parseInt(amount)
+  if (!amountInt) {
+    message.reply(PROPER_AMOUNT_TEXT)
+    return
+  }
+
+  // If amount has a comma
+  if (amount.indexOf(',') >= 0) {
+    message.reply(NO_COMMA_TEXT)
+    return
+  }
+
+  if (!toAddress) {
+    message.reply(NEED_ADDRESS_TEXT)
+    return
+  }
```

Comment:
Should we verify that it would be a proper Dogecoin address ? Or is the `sendFrom` command would reject this if not a proper dogecoin address ?